src/components/button/button.styles.ts

```diff
@@ -505,7 +505,10 @@ export default css`
   }
 
   .button--loading sl-spinner {
-    --indicator-color: currentColor;
+    --track-color: rgb(147 153 158 / 35%);
```

Comment:
I don't love hard-coding this color in here. Is there a way we can use a css variable instead?

Reply:
Oh, right! I forgot that we now have `rgb` variables for all the colors. I fixed to use the variable. TY for the callout!